lib/sqlalchemy/ext/asyncio/engine.py

```diff
@@ -879,7 +886,9 @@ async def do_something_async(async_engine: AsyncEngine) -> None:
 
         """  # noqa: E501
 
-        return await greenlet_spawn(fn, self._proxied, *arg, **kw)
+        return await greenlet_spawn(
+            fn, self._proxied, *arg, _require_await=False, **kw
```

Comment:
`_require_await` needs to be specified here?

Reply:
At runtime it is not required, but Mypy complains about it. If it is not an issue, I will put an ignore comment.